rfps/rfp-fvm.md

```diff
@@ -0,0 +1,112 @@
+# FVM Tooling & Infrastructure RFP
+
+## Overview
+
+We are seeking proposals for various developer and infrastructure tooling to extend the functionality and usability of the [Filecoin Virtual Machine](https://fvm.filecoin.io) (FVM).
+
+The Filecoin Virtual Machine (FVM) aims to combine smart contracts with co-location of storage and compute capabilities.
+
+The FVM is a WASM-based polyglot execution environment for IPLD data. It is designed to support native Filecoin actors written in languages that compile to WASM, as well as smart contracts written for foreign runtimes including the Ethereum Virtual Machine (EVM), Secure EcmaScript (SES), and eBPF.
+
+Refer to the resources section for more information on the FVM reference and specification, details on the possibilities FVM will unlock and the current roadmap.
+
+You can also consider contributing via the open [FVM Bug Bounties page here](https://immunefi.com/bounty/filecoin/)
+
+## Description
+
+The FVM is an integral and high-priority project for extending the functionality of Filecoin by bringing storage solutions to the blockchain and executing computation jobs or data pipelines on globally distributed data by pushing compute to the edges and coordinating its execution.
+
+Examples of future ideas include: async programming via continuations/futures, capability-orientation, time-locked transactions (using external beacons), parallel execution, native primitives for cross-chain interoperability, computation correctness proofs, and more.
```

Comment:
@DeveloperAlly - This paragraph is about the longer-term future of the FVM and is a bit confusing in the context of this RFP description for infra and dev tooling. I recommend removing it.

Reply:
I thought it might be good for context. As most of the grants will probably come through us anyway. I'm also fine to just remove it

Reply:
Removed